test/core/rules/L001_test.sqlflufftest

```diff
@@ -0,0 +1,5 @@
+RULE: L001
+
+TESTCASES:
+
+TEST1:
```

Comment:
Playing around with ideas for how we can make it easier and more readable to write multi line tests.

Reply:
I think if you wanted to go down that route you could represent the RuleTestCase in YAML files and have multiline SQL with 

```yaml
fail_str: |
  SELECT ... FROM ...
```

Maybe that's better than trying to figure out indentation when instantiating the rule test case class ?

Reply:
Yes this is just what I was looking for. Thank you

Reply:
Found this useful reference to explain the syntax (new to me!) https://yaml-multiline.info/